TruckMuncher/MapViewController.xib

```diff
@@ -1,35 +1,155 @@
 <?xml version="1.0" encoding="UTF-8" standalone="no"?>
-<document type="com.apple.InterfaceBuilder3.CocoaTouch.XIB" version="3.0" toolsVersion="7531" systemVersion="14D98g" targetRuntime="iOS.CocoaTouch" propertyAccessControl="none" useAutolayout="YES" useTraitCollections="YES">
+<document type="com.apple.InterfaceBuilder3.CocoaTouch.XIB" version="3.0" toolsVersion="7531" systemVersion="14D136" targetRuntime="iOS.CocoaTouch" propertyAccessControl="none" useAutolayout="YES" useTraitCollections="YES">
     <dependencies>
         <plugIn identifier="com.apple.InterfaceBuilder.IBCocoaTouchPlugin" version="7520"/>
     </dependencies>
     <objects>
         <placeholder placeholderIdentifier="IBFilesOwner" id="-1" userLabel="File's Owner" customClass="MapViewController" customModule="TruckMuncher" customModuleProvider="target">
             <connections>
-                <outlet property="mapView" destination="0bU-Q8-5aJ" id="fxP-8h-CFh"/>
-                <outlet property="view" destination="i5M-Pr-FkT" id="sfx-zR-JGt"/>
+                <outlet property="btnLinkFb" destination="jVw-CG-1cf" id="YDS-dS-7kl"/>
+                <outlet property="btnLinkTw" destination="Lzv-8e-Jfp" id="czS-sd-wWr"/>
+                <outlet property="lblPostToFb" destination="smL-Lj-Fqm" id="aqL-9U-gTw"/>
+                <outlet property="lblPostToTw" destination="7Zz-fg-vZy" id="l47-F3-zNn"/>
+                <outlet property="mapView" destination="R70-67-xFs" id="dY4-g2-PAw"/>
+                <outlet property="switchPostToFb" destination="Kst-pv-0Th" id="vYU-NH-zlj"/>
+                <outlet property="switchPostToTw" destination="9l6-mN-8mV" id="htx-do-RP9"/>
+                <outlet property="topMapConstraint" destination="gMe-WX-pNe" id="ENn-bP-upN"/>
+                <outlet property="view" destination="iN0-l3-epB" id="5RY-3t-i8j"/>
             </connections>
         </placeholder>
         <placeholder placeholderIdentifier="IBFirstResponder" id="-2" customClass="UIResponder"/>
-        <view clearsContextBeforeDrawing="NO" contentMode="scaleToFill" id="i5M-Pr-FkT">
-            <rect key="frame" x="0.0" y="64" width="600" height="600"/>
+        <view contentMode="scaleToFill" id="iN0-l3-epB">
+            <rect key="frame" x="0.0" y="0.0" width="600" height="600"/>
             <autoresizingMask key="autoresizingMask" widthSizable="YES" heightSizable="YES"/>
             <subviews>
-                <mapView clipsSubviews="YES" multipleTouchEnabled="YES" contentMode="scaleToFill" mapType="standard" pitchEnabled="NO" showsUserLocation="YES" showsBuildings="NO" showsPointsOfInterest="NO" translatesAutoresizingMaskIntoConstraints="NO" id="0bU-Q8-5aJ">
```

Comment:
I think we want to show the user location on the map. Looks like that got turned off in the xib